app/src/main/res/layout/activity_main.xml

```diff
@@ -43,62 +43,57 @@
             android:id="@+id/scrollview"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:fillViewport="true"
             android:background="#242424"
             android:orientation="horizontal">
-
             <LinearLayout
                 android:id="@+id/container"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:orientation="horizontal"
                 android:padding="8dp">
 
                 <com.google.android.material.button.MaterialButton
                     android:id="@+id/btn_smali"
-                    android:layout_width="wrap_content"
+                    android:layout_width="0dp"
```

Comment:
Hey, can you change this to use wrap_content back? Because using 0dp would reduce the readability

Reply:
okay